README.md

```diff
@@ -17,14 +17,4 @@ https://github.com/ztkmkoo/dss/blob/develop/CODE_CONVENTION.md
 
 ## CODE OF CONDUCT
 
-https://github.com/ztkmkoo/dss/blob/develop/CODE_OF_CONDUCT.md
```

Comment:
Not delete this..

Reply:
Sorry. i will fix it.

Reply:
Please check last line... the code is not deleted..

Reply:
oh.. you are right...
i will merge it now